.changeset/chatty-foxes-remember.md

```diff
@@ -0,0 +1,11 @@
+---
+'lit': patch
+'lit-element': patch
+'lit-html': patch
+'@lit/lit-starter-js': patch
+'@lit/lit-starter-ts': patch
+'@lit/localize': patch
+'@lit/reactive-element': patch
+---
+
+[internal] Consistently use a longer timeout when running tests.
```

Comment:
Does this timeout change require a public version bump?

Reply:
I don't think it does, but the changesets github check fails if there isn't a changeset file. I suppose I should just let it fail though :)